src/Core/Components/Checkbox/FluentCheckbox.razor.cs

```diff
@@ -0,0 +1,240 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection.Metadata;
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+using Microsoft.JSInterop;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// The FluentCheckbox component is used to render a checkbox input
+/// </summary>
+public partial class FluentCheckbox : FluentInputBase<bool>, IFluentComponentElementBase
+{
+
+    /// <inheritdoc />
```

Comment:
You need to add `cref`: `/// <inheritdoc cref="IFluentComponentElementBase.Element" />`